### PR TITLE
 adding Board,utility methods and testcase for Board

### PR DESCRIPTION
**Pull request by nik-1207**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import React from "react";
 import "./App.css";
```
Comment (Parm-S) [inline src/App.tsx:2]: Remove app.css

File: src/components/Board.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { formatNumber, LIMIT } from "../util/utility";
+
+const Board: React.FC = () => {
+  const arr: Array<number> = React.useMemo(
+    () => Array.from({ length: LIMIT }, (_) => 0),
+    []
+  );
+
+  return (
+    <Grid
```
Comment (Parm-S) [inline src/components/Board.tsx:12]: wrap it in paper

File: src/App.tsx (hunk 1 above)
Comment (nik-1207) [reply src/App.tsx:2]: it is used for theme i.e to center the number

Comment (Parm-S) [reply src/App.tsx:2]: use material-ui libraray
makeStyles

File: src/components/Board.tsx (hunk 2 above)
Comment (nik-1207) [reply src/components/Board.tsx:12]: should I do like this 
`<Paper><Grid><Grid/><Paper>`?

File: src/App.tsx (hunk 1 above)
Comment (nik-1207) [reply src/App.tsx:2]: will it be used after theme provider implementation?

Comment (Parm-S) [reply src/App.tsx:2]: you can use any time for alignment and all...

File: src/components/Board.tsx (hunk 2 above)
Comment (Parm-S) [reply src/components/Board.tsx:12]: it's optional...u can use it so its organize in a particular section

File: src/util/utility.test.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { formatNumber } from "./utility";
+
+describe("testinjg utility methods", () => {
```
Comment (AbhayMaheshwari123) [inline src/util/utility.test.ts]: testing spelling

**Review by AbhayMaheshwari123 — COMMENTED** [review]
a single change

**Review by Parm-S — CHANGES_REQUESTED** [review]

File: src/util/utility.test.ts (hunk 3 above)
Comment (nik-1207) [reply src/util/utility.test.ts]: resolved

**Pull request merged**